builder/builder.go

```diff
@@ -101,6 +96,15 @@ func NewBuilder(sk *bls.SecretKey, ds flashbotsextra.IDatabaseService, relay IRe
 }
 
 func (b *Builder) Start() error {
+	// Start regular payload attributes updates
+	go func() {
+		c := make(chan types.BuilderPayloadAttributes)
+		go b.beaconClient.SubscribeToPayloadAttributesEvents(c)
+		for {
```

Comment:
Is it time to add context handling within the builder?
It would be nice if `Stop` actually stopped this for loop